src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,49 @@
+.calendar {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  width: $width;
+  padding: 10px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  transform: translate(-50%, -50%);
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #eee;
+    width: $cell-size;
+    height: $cell-size;
+    box-sizing: border-box;
+    border: 1px solid black;
+    cursor: pointer;
+    transition: 0.5s;
```

Comment:
It is recommended to add transition properties for better performance.
Also, would be great to create a variable for transition-duration
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/974664cd-3a09-47a3-8933-ecbf76da95e1)

```suggestion
    transition: background-color 0.5s, transform 0.5s;
```
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x